companies/zucchetti.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "zucchetti",
+    "relevant-countries": [
+        "it",
+        "us",
+        "spa",
+        "br",
+        "fr"
+    ],
+    "categories": [
+        "it",
+        "commerce"
+    ],
+    "name": "Zucchetti",
+    "address": "Via Solferino 1, 26900 Lodi - Italia",
+    "phone": "+39 0371 594-1",
+    "email": "zucchettispa@gruppozucchetti.it",
```

Comment:
When possible, we use the contact details for privacy-related questions. They can usually be found in the privacy policy. In [this case](https://www.zucchetti.it/website/cms/privacy.html), there is a better email listed.